api-docs-slate/source/includes/_wallet_tx.md

```diff
@@ -114,24 +125,35 @@ let id;
 ```
 
 ```shell--vars
-id='foo'
+id='primary'
 ```
 
 ```shell--cli
 bwallet-cli --id=$id history
 ```
 
 ```shell--curl
-curl $url/wallet/$id/tx/history
+curl $walleturl/$id/tx/history
 ```
 
 ```javascript
-const wallet = WalletClient.wallet(id);
+const {WalletClient} = require('bclient');
+const {Network} = require('bcoin');
+const network = Network.get('regtest');
+
+const walletOptions = {
+  network: network.type,
+  port: network.walletPort,
+  apiKey: 'api-key'
+}
+
+const walletClient = new WalletClient(walletOptions);
+const wallet = walletClient.wallet(id);
```

Comment:
You may have already had this somewhere and I missed it, but I think we should document the difference between `walletClient.wallet(id)` version of the client and the client where you pass in the id as an argument like: `walletClient.getInfo('id')`. This is new and it tripped me up at first so it's a little confusing but also super useful to have both available (so you can create different clients for different wallets, or just use one walletClient to access different wallets depending on the id you pass it). it also makes the function signature slightly different depending on if you have the `walletClient.wallet` client or the normal `walletClient` client.

Reply:
added to #the-walletdb-and-object